mkosi/resources/man/mkosi.1.md

```diff
@@ -1260,6 +1260,7 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     | `ca-certificates`       | ✓      | ✓      | ✓      | ✓    | ✓      | ✓    | ✓        |
     | `coreutils`             | ✓      | ✓      | ✓      | ✓    | ✓      | ✓    | ✓        |
     | `cpio`                  | ✓      | ✓      | ✓      | ✓    | ✓      | ✓    | ✓        |
+    | `createrepo_c`          | ✓      | ✓      |        |      |        | ✓    | ✓        |
```

Comment:
Why not include it in Debian derivatives as well? The package is named [createrepo-c](https://packages.debian.org/bookworm/createrepo-c) or am I misunderstanding something and this is a different thing?

Reply:
Turns out we we're already installing it